src/renderer/marketplace/incorporation/detail-view/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import injectSheet from 'react-jss';
```

Comment:
@andregoncalves is there a need to react-jss here?

Reply:
Nice catch @rodrigopavezi , some old incorporations code still used react-jss.
I ended up refactoring and making the same change in other components as well.

Reply:
@andregoncalves awesome. cheers